.changeset/fix-cy-should.md

```diff
@@ -0,0 +1,5 @@
+---
+"@bigtest/cypress": patch
```

Comment:
```suggestion
"@bigtest/interactor": patch
```

Reply:
Oh, yes. Exactly ) I missed, that I made changes in interactors too